packages/react/src/components/CardEditor/CardEditForm/CardEditFormItems/TableCardFormItems/TableCardFormContent.jsx

```diff
@@ -331,7 +332,9 @@ const TableCardFormContent = ({
       >
         <ComboBox
           // need to re-gen if selected card changes or if a dataItem is removed from the list
-          key={`data-item-select-${removedDataItems.length}-selected_card-id-${cardConfig.id}`}
+          key={`data-item-select-${hash(validDataItemsForDropdown || {})}-selected_card-id-${
```

Comment:
Are we trying to use key to re-render if validDataItemsForDropdown changes? Do we have to hash this or can we just pass in the obj?

Reply:
can perhaps be replaced by `(() => obj)()`